tests/testthat/test-independent-testing-outdata.R

```diff
@@ -0,0 +1,47 @@
+library(dplyr)
+library(metalite)
+
+test_that("Its class is 'outdata'", {
+meta <- meta_dummy()
+output<-outdata(meta_dummy(), "apat", "wk12", "rel", n = meta$data_population %>% group_by(TRTA) %>% summarize(n = n()), group = "TRTA", reference_group = 1, order = 1:3)
+
+  expect_equal(class(output), "outdata")
+})
+
+
```

Comment:
Hi @burlave , thanks for the tests! It looks nice! I only have one minor comment: it looks like you missed the second bullet in the test plan of `outdata` (see https://github.com/Merck/metalite/issues/13#issue-1267752933)